ED/src/init/ed_type_init.f90

```diff
@@ -168,9 +169,16 @@ subroutine init_ed_cohort_vars(cpatch,ico, lsl,mzg,ntext_soil)
          cpatch%sla   (ico) = SLA(ipft)
          !---------------------------------------------------------------------------------!
       end select
+      ! Always use tabulated values for rd_bar
+      ! XX->MLO Check whether this is OK for your trait economics scenario
```

Comment:
@xiangtaoxu I think this is consistent with what was done before. Trait economics shouldn't matter here, the select case block above is for the light-controlled phenology.

An alternative would be to set the following:
```
catch%rd_bar(ico) = Rd0(ipft) * catch%vm_bar(ico) / Vm0 (ipft)
```
but I don't think this would be consistent with the light phenology paper.